== TEST (2019-05-29 18:14:39) 
-- CODE:
from numpy import*

n = int(input("tamanho do vetor: "))

i = 0
v = arange(n)

print (v)
-- ERROR:
Traceback (most recent call last):
  File "XXXX", line 1, in <module>
    from numpy import*
ModuleNotFoundError: No module named 'numpy'
*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*
== TEST (2019-05-29 18:18:22) 
-- CODE:
from numpy import*

n = int(input("tamanho do vetor: "))

i = 0
v = arange(n)
v = array([0, 1, 1])


print (v)
-- ERROR:
Traceback (most recent call last):
  File "XXXX", line 1, in <module>
    from numpy import*
ModuleNotFoundError: No module named 'numpy'
*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*
== TEST (2019-05-29 18:21:04) 
-- CODE:
from numpy import*

n = int(input("tamanho do vetor: "))

i = 0
v = zeros(n, dtype=int)



print (v)
-- ERROR:
Traceback (most recent call last):
  File "XXXX", line 1, in <module>
    from numpy import*
ModuleNotFoundError: No module named 'numpy'
*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*
== TEST (2019-05-29 18:21:04) 
-- CODE:
from numpy import*

n = int(input("tamanho do vetor: "))

i = 0
v = zeros(n, dtype=int)



print (v)
-- ERROR:
Traceback (most recent call last):
  File "XXXX", line 1, in <module>
    from numpy import*
ModuleNotFoundError: No module named 'numpy'
*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*
== TEST (2019-05-29 18:24:01) 
-- CODE:
from numpy import*

n = int(input("tamanho do vetor: "))
x = 1
i = 0
v = zeros(n, dtype=int)
while(i < size(v)):
	v.replace(0, x)
	x = x + x
	i = i + 1


print (v)
-- ERROR:
Traceback (most recent call last):
  File "XXXX", line 1, in <module>
    from numpy import*
ModuleNotFoundError: No module named 'numpy'
*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*
== TEST (2019-05-29 18:24:02) 
-- CODE:
from numpy import*

n = int(input("tamanho do vetor: "))
x = 1
i = 0
v = zeros(n, dtype=int)
while(i < size(v)):
	v.replace(0, x)
	x = x + x
	i = i + 1


print (v)
-- ERROR:
Traceback (most recent call last):
  File "XXXX", line 1, in <module>
    from numpy import*
ModuleNotFoundError: No module named 'numpy'
*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*
== TEST (2019-05-29 18:24:18) 
-- CODE:
from numpy import*

n = int(input("tamanho do vetor: "))
x = 1
i = 0
v = zeros(n, dtype=int)
while(i < size(v)):
	v.replace(0, x)
	x = x + x
	i = i + 1


print (v)
-- ERROR:
Traceback (most recent call last):
  File "XXXX", line 1, in <module>
    from numpy import*
ModuleNotFoundError: No module named 'numpy'
*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*
== TEST (2019-05-29 18:24:48) 
-- CODE:
from numpy import*

n = int(input("tamanho do vetor: "))
x = 1
i = 0
v = zeros(n, dtype=int)
while(i < size(v)):
	v.replace(0, x)
	x = x + x
	i = i + 1


print (v)
-- ERROR:
Traceback (most recent call last):
  File "XXXX", line 1, in <module>
    from numpy import*
ModuleNotFoundError: No module named 'numpy'
*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*
== TEST (2019-05-29 18:40:19) 
-- CODE:
from numpy import*

n = int(input("tamanho do vetor: "))
x = 1
i = 0
v = zeros(n, dtype=int)
while(i < size(v)):
	v[i] = x + x
	


print (v)
-- ERROR:
Traceback (most recent call last):
  File "XXXX", line 1, in <module>
    from numpy import*
ModuleNotFoundError: No module named 'numpy'
*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*
== TEST (2019-05-29 18:40:27) 
-- CODE:
from numpy import*

n = int(input("tamanho do vetor: "))
x = 1
i = 0
v = zeros(n, dtype=int)
while(i < size(v)):
	v[i] = x + x
	i = i + 1


print (v)
-- ERROR:
Traceback (most recent call last):
  File "XXXX", line 1, in <module>
    from numpy import*
ModuleNotFoundError: No module named 'numpy'
*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*
== TEST (2019-05-29 18:41:35) 
-- CODE:
from numpy import*

n = int(input("tamanho do vetor: "))
x = 0
i = 0
j = 0
a = 1
v = zeros(n, dtype=int)
while(i < size(v)):
	v[i] = x + j
	x = x + a
	i = i + 1


print (v)
-- ERROR:
Traceback (most recent call last):
  File "XXXX", line 1, in <module>
    from numpy import*
ModuleNotFoundError: No module named 'numpy'
*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*
== TEST (2019-05-29 18:45:29) 
-- CODE:
from numpy import*

n = int(input("tamanho do vetor: "))

x = 0
i = 0
a = 0
v = zeros(n, dtype=int)
while(i < size(v)):
	v[i] = x + a
	x = x + a
	a = a + 1
	i = i + 1

print (v)
-- ERROR:
Traceback (most recent call last):
  File "XXXX", line 1, in <module>
    from numpy import*
ModuleNotFoundError: No module named 'numpy'
*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*
== TEST (2019-05-29 18:45:46) 
-- CODE:
from numpy import*

n = int(input("tamanho do vetor: "))

x = 0
i = 0
a = 0
v = zeros(n, dtype=int)
while(i < size(v)):
	v[i] = x + a
	x = x + a
	a = a + 1
	i = i + 1

	print (v)
-- ERROR:
Traceback (most recent call last):
  File "XXXX", line 1, in <module>
    from numpy import*
ModuleNotFoundError: No module named 'numpy'
*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*
== TEST (2019-05-29 18:46:10) 
-- CODE:
from numpy import*

n = int(input("tamanho do vetor: "))

x = 0
i = 0
a = 0
v = zeros(n, dtype=int)
while(i < size(v)):
	v[i] = x + a
	x = x + a
	a = 1
	i = i + 1

	print (v)
-- ERROR:
Traceback (most recent call last):
  File "XXXX", line 1, in <module>
    from numpy import*
ModuleNotFoundError: No module named 'numpy'
*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*
== TEST (2019-05-29 18:46:43) 
-- CODE:
from numpy import*

n = int(input("tamanho do vetor: "))

x = 0
i = 0
a = 0
v = zeros(n, dtype=int)
while(i < size(v)):
	v[i] = x + a
	a = 1 + x
	i = i + 1

	print (v)
-- ERROR:
Traceback (most recent call last):
  File "XXXX", line 1, in <module>
    from numpy import*
ModuleNotFoundError: No module named 'numpy'
*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*
== TEST (2019-05-29 18:55:48) 
-- CODE:
from numpy import*

n = int(input("tamanho do vetor: "))

x = -2
i = 0
a = -1
v = zeros(n, dtype=int)
while(i < size(v)):
	v[i] = v[x] + v[a]
	v[1] = 1
	i = i + 1
	x = x + 1
	a = a + 1
print (v)
-- ERROR:
Traceback (most recent call last):
  File "XXXX", line 1, in <module>
    from numpy import*
ModuleNotFoundError: No module named 'numpy'
*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*
== TEST (2019-05-29 18:56:27) 
-- CODE:
from numpy import*

n = int(input("tamanho do vetor: "))

x = -2
i = 0
a = -1
v = zeros(n, dtype=int)
while(i < size(v)):
	v[i] = v[x] + v[a]
	v[1] = 1
	i = i + 1
	x = x + 1
	a = a + 1
	print (v)
-- ERROR:
Traceback (most recent call last):
  File "XXXX", line 1, in <module>
    from numpy import*
ModuleNotFoundError: No module named 'numpy'
*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*
== SUBMITION (2019-05-29 18:59:43) 
-- CODE:
from numpy import*

n = int(input("tamanho do vetor: "))

x = -2
i = 0
a = -1
v = zeros(n, dtype=int)
while(i < size(v)):
	v[i] = v[x] + v[a]
	v[1] = 1
	i = i + 1
	x = x + 1
	a = a + 1
print (v)
-- EXECUTION TIME:
0.53362
-- TEST CASE 1:
---- input:
10
---- correct output:
[ 0 1 1 2 3 5 8 13 21 34]
---- user output:
[ 0  1  1  2  3  5  8 13 21 34]
-- TEST CASE 2:
---- input:
8
---- correct output:
[ 0 1 1 2 3 5 8 13]
---- user output:
[ 0  1  1  2  3  5  8 13]
-- TEST CASE 3:
---- input:
18
---- correct output:
[ 0 1 1 2 3 5 8 13 21 34 55 89 144 233 377 610 987 1597]
---- user output:
[   0    1    1    2    3    5    8   13   21   34   55   89  144  233
  377  610  987 1597]
-- GRADE:
100%
*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*
